src/jit/gentree.h

```diff
@@ -2334,10 +2334,85 @@ struct GenTreeColon: public GenTreeOp
         {}
 };
 
-/* gtCall   -- method call      (GT_CALL) */
+// gtCall   -- method call      (GT_CALL)
 typedef class fgArgInfo *  fgArgInfoPtr;
 enum class InlineObservation;
 
+// Multi-reg return type descriptor of a GT_CALL node.
+// x64 Unix, Arm64, Arm32 and x86 allow a value to be returned in multiple
+// registers. For such calls this struct provides the following info 
+// on their return type
+//    - type of value returned in each return register
+//    - return register numbers in which the value is returned
+//    - a spill mask for lsra/codegen purpose
+//
+// Right now it is implemented to meet the needs of x64 unix.
+// Eventually it will be updated to meet the needs of Arm32/arm64.
+struct MultiRegReturnTypeDesc
+{
+private:
+    var_types regType0;
+    var_types regType1;
+
+#ifdef DEBUG
+    bool m_inited;
+#endif
+
+public:
+    MultiRegReturnTypeDesc()
+    {
+        regType0 = TYP_UNKNOWN;
+        regType1 = TYP_UNKNOWN;
+
+#ifdef DEBUG
+        m_inited = false;
+#endif
+    }
+
```

Comment:
IMO, all ctors use use initializer syntax for simple initialization, where it can be used (e.g., no control flow)

---
In reply to: [58250312](https://github.com/dotnet/coreclr/pull/4042#discussion_r58250312) [](ancestors = 58250312)